content/content.md

```diff
@@ -74,62 +74,57 @@ This process works like that for any repeating line. Give it a go, try drawing y
 
-Now at the start, I said it splits things into sine waves. The thing is, the sine waves it creates are not just regular sine waves, but they’re 3D. You could call them "complex sinusoids". Or just "spirals".
+首先，前面介紹過傅立葉轉換能將事物分析成多個正弦波。實際上，它產生出來的正弦波並不是一般的正弦波，而是3D(立體)的波形。也能稱為「複雜的正弦曲線」或「螺旋」。
 
 <canvas id="complex-sinusoid" class="sketch" width=500 height=500></canvas>
 
-If we take a look from the side, they look like sine waves. From front on, though, these look like circles.
+如果我們從側面看，它們看起來就像正弦波。從正面看，這些看起來像圓圈。
 
 <canvas id="complex-sinusoid-turn" class="sketch" width=500 height=500></canvas>
 
-So far everything we’ve been doing has only required the regular 2D sine waves. When we do a Fourier transform on 2D waves, the complex parts cancel out so we just end up with sine waves.
-
-But we can use the 3D sine waves to make something fun looking like this:
+目前為止，我們所做的一切只需要常規的2D(平面)正弦波。當我們對2D正弦波進行傅立葉轉換時，「複雜的」部分被忽略了，所以我們最終得到了正弦波。
+但是我們可以使用3D正弦波來使事情變得有趣，就像這樣：
 
 <canvas id="peace-epicycles" class="sketch" width=500 height=500></canvas>
 
-What’s going on here?
+這裡發生了什麼事情呢？
 
-Well, we can think of the drawing as a 3D shape because of the way it moves around in time. If you imagine the hand being drawn by a person, the three dimensions represent where the tip of their pencil is at that moment. The x and y dimensions tell us the position, and then the time dimension is the time at that moment.
+我們可以將一個手繪圖理解成一個3D的形狀，因為點的位置在隨著時間移動。如果您想像一個人證在繪製一隻手的圖形，那麼這三個維度就代表了某一個時刻中，鉛筆筆尖的位置。因此，除了 x 和 y 維度告訴我們筆尖得位置，還有一個 z 時間維度，構成了這樣的一個圖形。
 
 <canvas id="peace-3d" class="sketch" width=500 height=500></canvas>
 
-Now that we have a 3D pattern, we can't use the regular 2D sine waves to represent it. No matter how many of the 2D sine waves we add up, we'll never get something 3D. So we need something else.
+現在我們有了3D圖形，我們就不能使用一般2D正弦波來表示它。無論有多少個2D正弦波相加，我們都永遠不會得到3D。所以，我們還需要其他東西。
 
-What we can use is the 3D spiral sine waves from before. If we add up lots of those, we can get something that looks like our 3D pattern.
+我們可以使用的是之前的3D螺旋正弦波。如果我們將很多個螺旋相加，得到的結果就會看起來像是3D的形狀。
 
-Remember, these waves look like circles when we look at them from front on. The name for the pattern of a circle moving around another circle is an epicycle.
+請記住，當我們從正面看時，這些波形看起來像是圓圈。圍繞另一個圓圈運動的圓圈圖案，名稱就稱為「周轉園(均輪和本輪, Epicycle)」。
 
 <canvas id="peace-build-up" class="sketch" width=500 height=500></canvas>
 <input id="peace-build-up-slider" type="range" min="0" max="1" value="1" step="any">
 
-*Use the slider above to control how many circles there are.*
+*試著滑動看看，觀察到底有幾個圓圈。*
 
-Like before, we get a pretty good approximation of our pattern with just a few circles. Because this is a fairly simple shape, all the last ones do is make the edges a little sharper.
+如同前面敘述，只需要用幾個圓圈，就可以畫出很接近於原始的圖形。因為這是一個相當簡單的形狀，所以只要在最後添加小圓，就可以使邊緣更加精確。
 
-All this applies to any drawing, really! Now it’s your chance to play around with it.
+這個方法適用於任何一種圖案，現在，就是發揮您創作的機會啦！
 
 <div class="multi-container">
 <div class="sketch" >
```

Comment:
in the line: `<p id="draw-zone-instruction" class="instruction">Draw here!</p>`

Draw here! Also need to be translated

Just below that, in this line:
`<button id="draw-zone-undo-button" class="button embedded-button">Undo</button>`

Undo needs to be translated too